src/devices/halemeier.ts

```diff
@@ -88,4 +88,13 @@ export const definitions: DefinitionWithExtend[] = [
         fromZigbee: [fz.command_off, fz.command_on, fz.command_stop, fz.command_move],
         exposes: [e.action(["recall_*", "on", "off", "brightness_move_up", "brightness_move_down"])],
     },
+    {
+        zigbeeModel: ['HA-ZM12mw2-4K'],
+        model: 'HA-ZM12mw2-4K',
+        vendor: 'Halemeier',
+        description: '4-channel LED driver',
+        extend: [m.light({colorTemp: {range: [160, 450]}})],
```

Comment:
```suggestion
        extend: [m.light({colorTemp: {range: [160, 450]}, endpointNames: ['l1', 'l2', 'l3', 'l4']}), m.deviceEndpoints({endpoints: {l1: 1, l2: 2, l3: 3, l4: 4}})],
```

and remove `endpoint` and `meta`